src/components/types/index.tsx

```diff
@@ -28,25 +28,6 @@ const GeneralRecordPropType = PropTypes.shape({
   ...BASE_PROPERTIES,
 });
 
-/**
- * Represents general format of a statement class record
- */
-const StatementPropType = PropTypes.shape({
-  ...BASE_PROPERTIES,
-  conditions: PropTypes.arrayOf(PropTypes.shape({
-    ...BASE_PROPERTIES,
-  })),
-  evidence: PropTypes.arrayOf(PropTypes.shape({
-    ...BASE_PROPERTIES,
-  })),
-  relevance: PropTypes.shape({
-    ...BASE_PROPERTIES,
-  }),
-  subject: PropTypes.shape({
-    ...BASE_PROPERTIES,
-  }),
-});
-
 const LocationPropType = PropTypes.shape({
```

Comment:
it's not exported but it is used within this file so it can't be removed

Reply:
It seemed unnecessary since I was trying to keep the diff to a minimum, and all the prop types are on there way out anyway
so the `HistoryPropType` will be removed soon too